plugins/builds/create.js

```diff
@@ -1,10 +1,12 @@
 'use strict';
+const async = require('async');
 const boom = require('boom');
+const hoek = require('hoek');
 const urlLib = require('url');
 const validationSchema = require('screwdriver-data-schema');
 const Model = require('screwdriver-models');
 
-module.exports = (server) => ({
+module.exports = (server, options) => ({
```

Comment:
Question for anyone (not just the submitters): do we want to bother with documenting the method signature here?